### ISSUE TITLE
alembic.ini template missing from 0.7.0b1 package

### ISSUE THREAD

**Issue by parente** [jupyterhub/jupyterhub#877]
Alembic template is not included in the 0.7.0b1 pypi package.

**How to reproduce the issue**

```
pip install --pre jupyterhub
cd /folder/with/my/jupyterhub_config.py
jupyterhub upgrade-db

[E 2016-11-21 12:23:01.216 JupyterHub app:1524]
    Traceback (most recent call last):
      File "/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/site-packages/jupyterhub/app.py", line 1522, in launch_instance_async
        yield self.start()
      File "/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/site-packages/jupyterhub/app.py", line 1433, in start
        self.subapp.start()
      File "/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/site-packages/jupyterhub/app.py", line 185, in start
        dbutil.upgrade(hub.db_url)
      File "/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/site-packages/jupyterhub/dbutil.py", line 78, in upgrade
        with _temp_alembic_ini(db_url) as alembic_ini:
      File "/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/contextlib.py", line 59, in __enter__
        return next(self.gen)
      File "/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/site-packages/jupyterhub/dbutil.py", line 66, in _temp_alembic_ini
        write_alembic_ini(alembic_ini, db_url)
      File "/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/site-packages/jupyterhub/dbutil.py", line 30, in write_alembic_ini
        with open(ALEMBIC_INI_TEMPLATE_PATH) as f:
    FileNotFoundError: [Errno 2] No such file or directory: '/Users/parente/miniconda3/envs/jupyterhub-0.7.0b1/lib/python3.5/site-packages/jupyterhub/alembic.ini'
```

**Share what version of JupyterHub you are using**

0.7.0b1

**Comment by yuvipanda** [comment]
Similar to #825

**Comment by parente** [comment]
Doh. Missed that somehow. Yes, looks like the same bug.

**Comment by willingc** [comment]
@yuvipanda @parente I'm wondering if this is a workflow documentation issue for JupyterHub instead of a bug. If you create an alembic environment (http://alembic.zzzcomputing.com/en/latest/tutorial.html#creating-an-environment), using:

-  `alembic init alembic`

Then generate an alembic.ini does the error still occur.

**Comment by minrk** [comment]
It's a bug in setup.py. The alembic.ini in the repo should be included in the package_data. Should be fixed by #882
